scapy/utils.py

```diff
@@ -1305,8 +1305,16 @@ def write(self, pkt):
             else:
                 pkt = pkt.__iter__()
             for p in pkt:
+
                 if not self.header_present:
                     self._write_header(p)
+
+                if self.linktype != conf.l2types.get(type(p), None):
+                    msg = "Inconsistent linktypes detected!"
+                    msg += " The resulting PCAP file might contain"
+                    msg += " invalid packets."
+                    warning(msg)
```

Comment:
```suggestion
                    warning(
                        "Inconsistent linktypes detected! "
                        "The resulting PCAP file might contain "
                        "invalid packets"
                    )
```
To avoid useless string objects creation.

Reply:
Done.